java/src/apps/ConfigBundle.java

```diff
@@ -22,7 +22,7 @@
  * @version $Revision: 17977 $
  * @since 3.3.1
  */
-public class ConfigBundle extends apps.Bundle {
+public class ConfigBundle extends jmri.Bundle {
```

Comment:
My last change didn't hang the Prefs panel on either of the apps. Before I adjusted the extend path, it did.